src/ConditionManager.cpp

```diff
@@ -61,16 +61,17 @@ bool ConditionManager::propagateHVPMTValue(std::size_t id) {
 }
 
 bool ConditionManager::propagateHVPMTState(std::size_t id) {
-    bool result;
-    bool state = getHVPMTSetState(id);
-    if (state)
-        result = m_setup_manager->switchHVPMTON(id);
+    if (getHVPMTSetState(id))
+        return m_setup_manager->switchHVPMTON(id);
     else
-        result = m_setup_manager->switchHVPMTOFF(id);
+        return m_setup_manager->switchHVPMTOFF(id);
```

Comment:
I don't remember the conclusion on that... So we don't do something that reads the actual state of the HV?